inc/deployfile.class.php

```diff
@@ -53,6 +53,7 @@ class PluginGlpiinventoryDeployFile extends PluginGlpiinventoryDeployPackageItem
     const REGISTRY_NO_DB_ENTRY = 0x1;
     const REGISTRY_NO_MANIFEST = 0x2;
 
+    const DL_MAX_SIZE = 1024 * 1024 * 50;
```

Comment:
I think we should not limit downloadable package size, even if package weights 10GB.
If `Toolbox::sendFile` does not handle it, I may have a fix for it.

Reply:
We need to unzip all parts and concatenate them.
I'm not confident for big files, and the process can take a lot of time before really starting the download (and so the user seing progression).
50MB is maybe a little too small, but i still think a limit should be set

Reply:
Maybe output can be flushed after each part processing.